test/fixtures/generator/bwx.xcodeproj/xcshareddata/xcschemes/generator.xcscheme

```diff
@@ -91,7 +91,7 @@
       selectedDebuggerIdentifier = "Xcode.DebuggerFoundation.Debugger.LLDB"
       selectedLauncherIdentifier = "Xcode.DebuggerFoundation.Launcher.LLDB"
       customLLDBInitFile = "$(BAZEL_LLDB_INIT)"
-      shouldUseLaunchSchemeArgsEnv = "YES">
+      shouldUseLaunchSchemeArgsEnv = "NO">
```

Comment:
What is the affect of this? Does this mean the `BAZEL_WORKSPACE_DIRECTORY` env isn't passed anymore? If so, we need that set explicitly for https://github.com/bazelbuild/apple_support/pull/128 to work.

Reply:
This unchecks the box in the Xcode dialog allowing the custom args/env vars to be used.

I will update the code to ensure that the bazel env vars are included.

Reply:
If the magic Bazel env vars need to be set in every BwB scheme, then the current logic for custom schemes is not quite right. It only includes them if the a launch action is defined. 

@brentleyjones Can you confirm that we do want the magic Bazel env vars set in the launch action for every BwB scheme?

Reply:
We want them set in both the launch and test actions, yes. While Bazel doesn't set `BAZEL_WORKSPACE_DIRECTORY` by default for tests, we still need it for the above feature I liked.

Reply:
Looks like this one doesn't have the bazel magic env variables still?

Reply:
This file is for BwX. I am seeing the magic env vars in the test action in the BwB version of the scheme.